app/addons/activetasks/components.react.jsx

```diff
@@ -0,0 +1,610 @@
+
+      return (
+        <div id="dashboard-lower-content">
+          <table className="table table-bordered table-striped active-tasks">
+            <ActiveTasksTableHeader 
+              onTableHeaderClick={onTableHeaderClick}
+              sortByHeader={sortByHeader}
+              headerIsAscending={headerIsAscending}/>
+            <ActiveTasksTableBody 
+              collection={collection} 
+              selectedRadio={selectedRadio} 
+              searchTerm={searchTerm}/>
+          </table>
+        </div>
+      );
+    }
+  });
+
+  var ActiveTasksTableHeader = React.createClass({
+    getDefaultProps: function () {
+      return {
+        headerNames : [
+          ['type', 'Type'],
+          ['database', 'Database'],
+          ['started_on', 'Started On'],
+          ['updated_on', 'Updated On'],
+          ['pid', 'PID'],
+          ['progress', 'Status']
+        ]
+      };
+    },
+
+    createTableHeadingFields: function () {
+      var onTableHeaderClick = this.props.onTableHeaderClick;
+      var sortByHeader = this.props.sortByHeader;
+      var headerIsAscending = this.props.headerIsAscending;
+      return (
+        this.props.headerNames.map(function (header) {
+          return (
+            <TableHeader 
+              HeaderName={header[0]}
+              DisplayName={header[1]}
+              key={header[0]}
+              onTableHeaderClick={onTableHeaderClick}
+              sortByHeader={sortByHeader}
+              headerIsAscending={headerIsAscending} />
+          );
+        })
+      );
+    },
+
+    render: function () {
+      var tableHeadingFields = this.createTableHeadingFields();
+      return (
+        <thead>
+          <tr>{tableHeadingFields}</tr>
+        </thead>
+      );
+    }
+  });
+
+  var TableHeader = React.createClass({
+    arrow: function () {
+      var sortBy = this.props.sortByHeader;
+      var currentName = this.props.HeaderName;
+      var headerIsAscending = this.props.headerIsAscending;
+      var arrow = headerIsAscending ? 'icon icon-caret-up' : 'icon icon-caret-down';
+
+      if (sortBy === currentName) {
+        return <i className={arrow}></i>;
+      }
+    },
+
+    onTableHeaderClick: function (e) {
+      var headerSelected = e.target.value;
+      this.props.onTableHeaderClick(headerSelected);
+    },
+
+    render: function () {
+      var arrow = this.arrow();
+      var th_class = 'header-field ' + this.props.HeaderName;
+
+      return (
+        <input
+          type="radio"
+          name="header-field"
+          id={this.props.HeaderName}
+          value={this.props.HeaderName}
+          className="header-field radio"
+          onChange={this.onTableHeaderClick}>
+          <th className={th_class} value={this.props.HeaderName}>
+            <label 
+              className="header-field label-text"
+              htmlFor={this.props.HeaderName}>
+              {this.props.DisplayName} {arrow}
```

Comment:
this.props. DisplayName -> this.props.DisplayName

Uppercase is reserved for constructor functions per convention in JS